coriolis/osmorphing/suse.py

```diff
@@ -127,6 +127,16 @@ def _has_systemd(self):
         except Exception:
             return False
 
+    def _configure_cloud_init(self):
+        super(BaseSUSEMorphingTools, self)._configure_cloud_init()
+        if self._has_systemd():
+            self._enable_systemd_service("cloud-init")
+
+    def post_packages_install(self, package_names):
+        self._run_dracut()
+        self._configure_cloud_init()
```

Comment:
NIT for consistency's sake:
```suggestion
        self._configure_cloud_init()
        self._run_dracut()
```